esp/esp/program/models/class_.py

```diff
@@ -1002,6 +1003,12 @@ def cancel(self, email_students=True, include_lottery_students=False, explanatio
                 else:
                     send_mail(ssi_email_title, msgtext, from_email, to_email)
 
+        if self.parent_program.hasModule('GroupTextModule') and text_students:
```

Comment:
Probably we should error or otherwise report back to the user if `text_students` is somehow `True`  but the module is not enabled!  (Alternately or in addition, we can just remove it from the form if the module isn't there?)